packages/@aws-cdk/aws-ivs/README.md

```diff
@@ -1,6 +1,5 @@
 # AWS::IVS Construct Library
 <!--BEGIN STABILITY BANNER-->
-
 ---
```

Comment:
since this now has L2 constructs, the stability of the module needs to be updated to `experimental` and the corresponding banner update needs to be added

Reply:
I think the stability already is experimental. Where can I find what changes need to be made to the banner?

Reply:
the `maturity` as well.
you can check out any other module which is currently example (i.e. `appsync`) for both the changes to `package.json` and the changes to the `README`

Reply:
link to appsync [README](https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-appsync)
and the [package.json](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-appsync/package.json#L121-L122)